roadservice/core/models.py

```diff
@@ -403,7 +510,37 @@ def accept_issue(self, issue, mission_type, speciality_requirements, machinery_r
 
         issue.state = Issue.State.ACCEPTED
         issue.save()
-        issue.assign_resources(mission_type)
+        return issue.assign_resources(mission_type)
 
-    def get_concrete(self):
-        return self
+    def reject_issue(self, issue):
+        if self.county == issue.county:
+            issue.state = Issue.State.REJECTED
+            issue.save()
+            return True
+        return False
+
+    def get_issues(self):
+        return self.county.issue_set.all()
+
+    def get_reported_issues(self):
+        return self.county.issue_set.filter(state=Issue.State.REPORTED)
+
+    @staticmethod
+    def get_mission_types():
```

Comment:
I don't think these should be static methods. I'll explain my thinking in person.